content/guides/references/configuration.md

```diff
@@ -822,20 +822,20 @@ DEBUG=cypress:cli,cypress:server:specs
 
 ## History
 
-| Version                                                                 | Changes                                                                              |
-| ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
-| [CFG_VERSION](/guides/references/changelog#CFG_VERSION_CHANGED_TO_HASH) | Reworked page to support new `cypress.config.js` and deprecated `cypress.json` files |
-| [8.7.0](/guides/references/changelog#8-7-0)                             | Added `slowTestThreshold` option                                                     |
-| [8.0.0](/guides/references/changelog#8-0-0)                             | Added `clientCertificates` option                                                    |
-| [7.0.0](/guides/references/changelog#7-0-0)                             | Added `e2e` and `component` options.                                                 |
-| [7.0.0](/guides/references/changelog#7-0-0)                             | Added `redirectionLimit` option.                                                     |
-| [6.1.0](/guides/references/changelog#6-1-0)                             | Added `scrollBehavior` option.                                                       |
-| [5.2.0](/guides/references/changelog#5-2-0)                             | Added `includeShadowDom` option.                                                     |
-| [5.0.0](/guides/references/changelog#5-0-0)                             | Added `retries` configuration.                                                       |
-| [5.0.0](/guides/references/changelog#5-0-0)                             | Renamed `blacklistHosts` configuration to `blockHosts`.                              |
-| [4.1.0](/guides/references/changelog#4-12-0)                            | Added `screenshotOnRunFailure` configuration.                                        |
-| [4.0.0](/guides/references/changelog#4-0-0)                             | Added `firefoxGcInterval` configuration.                                             |
-| [3.5.0](/guides/references/changelog#3-5-0)                             | Added `nodeVersion` configuration.                                                   |
+| Version                                      | Changes                                                                              |
+| -------------------------------------------- | ------------------------------------------------------------------------------------ |
+| [10.0.0](/guides/references/changelog#1000)  | Reworked page to support new `cypress.config.js` and deprecated `cypress.json` files |
```

Comment:
Will this actually be `#1000`? OR will it be `#10-0-0`?

Reply:
Weird. I followed a pattern that I came across for these previous release URLS but can't find where that was. I'll tweak these.

Reply:
Oh interesting. It's when you load this up on a localhost. The format is #930, etc.